nengo_dl/tests/test_benchmarks.py

```diff
@@ -242,10 +242,10 @@ def test_performance(net, train, minibatch_size, eager, min, max):
     # performance is based on Azure NC6 VM
     # CPU: Intel Xeon E5-2690 v3 @ 2.60Ghz
     # GPU: Nvidia Tesla K80
-    # Python version: 3.6.10
-    # TensorFlow GPU version: 2.3.0
+    # Python version: 3.7.1
+    # TensorFlow GPU version: 2.2.1
```

Comment:
Why did this regress?

Reply:
It was actually always 2.2 (because we were installing from `conda`, which lagged behind the latest `pip` release).

Reply:
Ok, I added this to the commit comment.